src/emulator/functionsEmulator.ts

```diff
@@ -527,6 +530,63 @@ You can probably fix this by running "npm install ${
     return loadTriggers();
   }
 
+  addRealtimeDatabaseTrigger(
+    projectId: string,
+    definition: EmulatedTriggerDefinition
+  ): Promise<any> {
+    const databasePort = EmulatorRegistry.getPort(Emulators.DATABASE);
+    if (!databasePort) {
+      EmulatorLogger.log(
+        "INFO",
+        `Ignoring trigger "${
+          definition.name
+        }" because the Realtime Database emulator is not running`
+      );
+      return Promise.resolve();
+    }
+    if (definition.eventTrigger === undefined) {
+      EmulatorLogger.log(
+        "WARN",
+        `Event trigger "${definition.name}" has undefined "eventTrigger" member`
+      );
+      return Promise.reject();
+    }
+    const bundle = JSON.stringify([
+      {
+        name: `projects/${projectId}/locations/_/functions/${definition.name}`,
+        path: definition.eventTrigger.resource.split("refs")[1],
```

Comment:
This is too hacky IMO. Also, this can break if `refs` literally appear in the path, like `/refs/foo/bar/refsomething/baz`.